packages/blocks/src/api/raw-handling/ms-strip-footnotes.js

```diff
@@ -0,0 +1,22 @@
+export default function( node ) {
```

Comment:
I think we can just merge these two files? Maybe something like `ms-strip-notes`.